src/UiModal.vue

```diff
@@ -190,12 +194,17 @@ export default {
             }
         },
 
-        onBackdropClick() {
-            if (this.dismissOnBackdrop) {
+        onBackdropMouseDown() {
+            this.backdropMouseDown = true;
```

Comment:
The boolean could be misread as indicating that the mouse is being held down on the backdrop. Perhaps this is less likely to be confused:

```suggestion
            this.mouseDownSource = 'backdrop';
```